tools/filters/wig_to_bigwig.xml

```diff
@@ -1,7 +1,7 @@
-<tool id="wig_to_bigWig" name="Wig/BedGraph-to-bigWig" version="1.1.1">
+<tool id="wig_to_bigWig" name="Wig/BedGraph-to-bigWig" version="1.1.2" profile="22.05">
```

Comment:
We shouldn't update build-in tools in such major ways. Can you export this to the IUC if there's no other tool doing this ?

Reply:
Thanks @mvdbeek - of course - if that's the preferred solution, will close this and move it to iuc tools. 
TBH I was puzzling over how to get a new test tested. Ain't no 2.wig anywhere I could find.